cores/arduino/ard_sup/ap3_analog.h

```diff
@@ -44,6 +44,7 @@ ap3_err_t ap3_change_channel(ap3_gpio_pad_t padNumber);
 bool power_adc_disable();
 uint16_t analogRead(uint8_t pinNumber);
 ap3_err_t analogReadResolution(uint8_t bits);
+float getTemperature();
```

Comment:
I feel like ```getTemperature()``` is just a bit too generic. What am I getting the temperature of? Should I be passing in an argument that is a value to convert to temperature?

How about something like ```getInternalTempC()``` or ```getChipTempC()```?